datanode/sqlstore/connection_source.go

```diff
@@ -271,3 +263,22 @@ func (t *delegatingConnection) QueryFunc(ctx context.Context, sql string, args [
 	}
 	return t.pool.QueryFunc(ctx, sql, args, scans, f)
 }
+
+func CreateConnectionPool(conf ConnectionConfig) (*pgxpool.Pool, error) {
```

Comment:
we set a minimum pool size for the network-history pool, but not, and never had, for the sqlstores pool (note: write connection is seperate and isolated) - should we?   sensible default?